lib/l10n/intl_it.arb

```
@@ -1324,5 +1324,12 @@
   "shareFeatureComingSoon": "Funzione di condivisione disponibile nella Fase 2",
   "devTools": "Strumenti di Sviluppo",
   "devToolsWarning": "Solo per debug e risoluzione problemi",
-  "viewAndExportLogs": "Visualizza ed esporta i log dell'applicazione"
+  "viewAndExportLogs": "Visualizza ed esporta i log dell'applicazione",
+  "saveLogs": "Salva Log",
+  "logsExportSuccess": "Log esportati con successo",
+  "logsExportError": "Impossibile esportare i log",
+  "shareLogsError": "Impossibile condividere i log",
+  "shareLogs": "Condividi Log",
```

RuboCop (1.82.1)

[warning] Duplicated key in hash literal.

(Lint/DuplicateHashKey)